docs/tr1/CHANGELOG.md

```diff
@@ -11,7 +11,8 @@
 - added exit fade-out effects (#2348)
 - added optional dynamic lighting for gun flashes and explosions, similar to TR2+ (#2357)
 - ⚠️ changed the game data to use a separate strings file for text information, removing it from the game flow file
-- changed the sprite limit from 512 to 1024
+- changed the sprite texture limit from 2048 to unlimited (within game's overall memory cap)
```

Comment:
@lahm86 @rr- this first TR1 changelog entry should say `object texture limit` I believe. I think it's wrong in `develop` currently.

Reply:
You're right. I can fix in the next texture PR. Thanks!